test/unit/builders/test_violators_avaliations_builder.py

```diff
@@ -1,115 +1,324 @@
 import unittest
 
+from datetime import datetime
+
 from main_penalty_calculate.builders import ViolatorsAvaliationsBuilder
 from main_penalty_calculate.models import (
     TrafficViolation,
     IdentityCard,
     LicensePlate,
     ViolatorAvaliation
 )
+from main_penalty_calculate.constants import TypeInfractionStrings
 
 
 class TestViolatorsAvaliationsBuilder(unittest.TestCase):
-    def test_build_violator_avaliation(self):
+    def test_build_violator_avaliation_with_different_identity_cards(self):
         traffic_violations = [
             TrafficViolation(
                 identity_card=IdentityCard('19.632.142-6', 'Takashi'),
-                license_plate=LicensePlate('IDE-3516')
+                license_plate=LicensePlate('IDE-3516'),
+                type_infraction=TypeInfractionStrings.VERY_SERIOUS,
+                infraction_date=datetime(1999, 10, 1, 6, 12, 22),
```

Comment:
Dica: como esse teste nao tem nada a ver com validacao da infraction date eu deixaria bem proximo assim como vc fez no outro, vc nao quer que quando a data for mudada esse testes quebre tambem.